extensions/blocks/tiled-gallery/tiled-gallery.php

```diff
@@ -26,14 +28,92 @@ class_exists( 'Jetpack_Photon' ) && Jetpack::is_module_active( 'photon' )
 	 *
 	 * @return string
 	 */
-	function jetpack_tiled_gallery_load_block_assets( $attr, $content ) {
```

Comment:
Is the removal of this function a concern? Is there any reason to maintain it just wrapping the static method?

```php
jetpack_tiled_gallery_load_block_assets( $attr, $content ) {
	Jetpack_Tiled_Gallery_Block::render( $attr, $content );
}
```

Reply:
>Is there any reason to maintain it just wrapping the static method?

Not that I know, seems safe to remove. 👍